pkg/controller/scheduling/scheduling.go

```diff
@@ -159,6 +159,10 @@ func ResourceRequirements(req router.Request, app *v1.AppInstance, containerName
 	}
 
 	requirements := &corev1.ResourceRequirements{Limits: corev1.ResourceList{}, Requests: corev1.ResourceList{}}
+	if z.Dereference(cfg.IgnoreResourceRequest) {
+		return requirements, nil
+	}
```

Comment:
This also causes us to skip non-memory related resource requirements, aka, the requirements that landed recently to enable GPU (this PR: https://github.com/acorn-io/runtime/pull/2384)


....I just need a minute to think this through

Reply:
Yeah, I think this is fine as-is. The alternative would be finer grained flags - one for memory and cpu, the other for generic "Resources", but I think that's overkill.